test/functional/lib/tests.cpp

```diff
@@ -4473,8 +4475,11 @@ GenericRxFragmentBuffer(
     //
     // Allocate UDP payload and initialize to a pattern.
     //
-    unique_malloc_ptr<UCHAR> Payload((UCHAR *)malloc(Params->PayloadLength));
-    CxPlatRandom(Params->PayloadLength, Payload.get());
+    unique_malloc_ptr<UCHAR> Payload;
+    if (Params->PayloadLength > 0) {
+        Payload.reset((UCHAR *)AllocMem(Params->PayloadLength));
```

Comment:
for this pattern, will the tests continue to throw errors if allocs fail?

Reply:
Nope, are you suggesting I add `TEST_NOT_NULL(Payload.get());`>

Reply:
maybe just add TEST_NOT_NULL to AllocMem

Reply:
I don't think we can do that in kernel mode. There's another PR that converts all exception throwing to returning false in the test function.